examples/active_qubit_reset.md

```diff
@@ -0,0 +1,518 @@
+---
+jupyter:
+  jupytext:
+    formats: ipynb,md
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.14.4
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+# Active Qubit Reset
+
+
+This tutorial shows how to perform an active qubit reset experiment with real-time feedback using Zurich Instruments PQSC, SHFQA and SHFSG devices.
+
+Active qubit reset is a procedure to initialize the qubit in the ground state by measuring its state: if the qubit is measured in the exited state, then a $\pi$ pulse is applied to bring it in the ground state, while if the qubit is measured in the ground state then it's already initialized and there's nothing else to be done. In this example we will use the SHFQA to perform the readout of the qubit's state, the SHFSG to apply the conditional resetting pulse, and the PQSC to coordinate the experiment, enabling real-time feedback pulse with as little latency as possible.
+
+Requirements:
+- LabOne Version >= 22.08
+- 1 SHFQA
+- 1 SHFSG
+- 1 PQSC
+- ZSync connections between the PQSC and the SHFQA and between the PQSC and the SHFSG
+- Loopback connection between input and output of channel 0 of the SHFQA
```

Comment:
better to use a cardinal number (first channel) to avoid ambiguities of 0 and 1 based indexing, respectively used in the API and on the front panel